src/main/java/io/supertokens/passwordless/PasswordlessLinkCodeHash.java

```diff
@@ -0,0 +1,22 @@
+package io.supertokens.passwordless;
+
+import java.util.Base64;
+
+public class PasswordlessLinkCodeHash {
+    private final String encodedValue;
+
+    public PasswordlessLinkCodeHash(byte[] bytes) {
+        // We never do anything further with the bytes, so we can just encode, store and reuse it.
+        // If we choose to do storage based on bytes this can change.
+        this.encodedValue = Base64.getUrlEncoder().encodeToString(bytes);
+    }
+
+    public PasswordlessLinkCodeHash(String encodedValue) {
+        this.encodedValue = encodedValue;
+        // We could do validation here...
+    }
```

Comment:
If this is not being used anywhere, we should remove this. Also, the comment indicates that something is missing.

Reply:
I just added that to mirror the one from DeviceIdHash. Validation is indeed something we could do, but I'm not sure if it's necessary.